apisix/plugins/response-rewrite.lua

```diff
@@ -85,7 +90,13 @@ do
 
 function _M.body_filter(conf, ctx)
     if conf.body then
-        ngx.arg[1] = conf.body
+
+        if conf.body_base64 then
+            ngx.arg[1] = ngx.decode_base64(conf.body)
```

Comment:
if conf.body is not well formed, will return `nil`. we need add test case for it.

Reply:
done